packages/api/tests/endpoints/executions.js

```diff
@@ -115,11 +150,68 @@ test('GET logs returns all executions', async (t) => {
 
 test('GET executions with asyncOperationId filter returns the correct executions', async (t) => {
   const response = await request(app)
-    .get('/executions?asyncOperationId=012345-12345')
+    .get('/executions?asyncOperationId=0fe6317a-233c-4f19-a551-f0f76071402f')
     .set('Accept', 'application/json')
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
   t.is(response.body.meta.count, 1);
   t.is(response.body.results[0].arn, 'arn2');
 });
+
+test('DELETE removes only specified execution from all data stores', async (t) => {
+  const {
+    executionDynamoModel,
+    executionPgModel,
+    testKnex,
+    executions,
+  } = t.context;
+
+  const newExecution = {
+    arn: 'arn3',
+    status: 'completed',
+    name: 'test_execution',
+  };
+
+  executions.push(newExecution);
+
+  await Promise.all(executions.map(async (execution) => {
+    delete execution.asyncOperationId;
```

Comment:
Oh so the reason I'm doing this is to avoid needing to create an Async Operation just to test execution deletion. If we have that `execution.asyncOperationId` and I go to create the PG record, I'll need to create a valid async operation with a matching ID. That isn't necessary for testing execution deletion here so I'm removing it.

Added a comment for clarification:
https://github.com/nasa/cumulus/pull/2252/commits/e886c0ce468fa396de61f53db59ec96963f73610

Reply:
Oops, no, that's my bad, I misread that as explicitly deleting the *record* not the object key.  🤦🏻        Thanks for the comment, in either case 👍🏻